sdk/communication/communication-short-codes/test/public/utils/recordedClient.ts

```diff
@@ -116,12 +118,12 @@ export const testPollerOptions = {
 };
 
 function createTestHttpClient(): HttpClient {
-  const customHttpClient = new DefaultHttpClient();
+  const customHttpClient = new ServiceClient();
```

Comment:
The replacement should be `createDefaultHttpClient()` from `@azure/core-rest-pipeline`